kernel/network/routing.c

```diff
@@ -80,37 +83,40 @@ arp_table_get_mac(uint32_t ip)
 }
 
 void
-arp_table_add_entry(mac_address_t mac, uint32_t ip) {
+arp_table_add_entry(mac_address_t mac, uint32_t ip)
+{
   uint32_t slot = 0;
 
   // find next free slot, if not existing overwrite oldest
-  for(uint32_t i = 0; i < MAX_ARP_TABLE_ENTRIES; i++)
+  for (uint32_t i = 0; i < MAX_ARP_TABLE_ENTRIES; i++)
     {
-      if(!arp_table[i].entry_time)
+      if (!arp_table[i].entry_time)
         {
           slot = i;
           break;
         }
-      else if(arp_table[i].entry_time < arp_table[slot].entry_time)
+      else if (arp_table[i].entry_time < arp_table[slot].entry_time)
         slot = i;
     }
-
-  arp_table[slot] = (arp_table_entry_t) {ip, mac, clock_seconds()};
+  /* *INDENT-OFF* */
+  arp_table[slot] = (arp_table_entry_t) { ip, mac, clock_seconds() };
+  /* *INDENT-ON* */
```

Comment:
What would the proposed indentation of `indent` be?

Reply:
```c
arp_table[slot] = (arp_table_entry_t)
{
ip, mac, clock_seconds()};
```

Reply:
Uff